### DISCUSSION TITLE
"now playing" indication

### DISCUSSION THREAD

**Discussion by ceewees** [dtcooper/raspotify#579] [Q&A] [answered]
Hello Jason,
In Album view, Spotify ui shows which song is currently playing by a "quasi animated rta/graph", but as soon as I switch to raspotify, I will loose that animation.
Is that something more for the librespot folks to look at?

**Comment by JasonLG1979** [comment, marked as answer]
Librespot is an unofficial client and the product of reverse engineering a non-public and undocumented API. We get no help from Spotify other than so far they have graciously decided not to blacklist us. It will probably always be a work-in-progress. As such there will be things here and there that may not be functional or may not work exactly like an official client.

**Reply by ceewees** [reply]
easy to forget we live by the grace of Spotify.
also, somewhere a scary thought though
thank you
